great-idea/services.html

```diff
@@ -1,87 +1,123 @@
-<!-- copy and paste your index.html page structure in here, markup the content provided after that -->
-
-
-Services
-Product
-Vision
-Features
-About
-Contact
-
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-
-
-<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
-
-Services
-
-Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+<!doctype html>
 
+<html lang="en">
+<head>
+  <meta charset="utf-8">
 
-Digital Design
+  <title>Great Idea! - Services</title>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
 
-Learn More
-
-
-
-UX / UI
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
 
+  <body>
 
+<!-- copy and paste your index.html page structure in here, markup the content provided after that -->
+    <div class="container">
+
+        <header>
+            <nav>
+              <a href="services.html">Services</a>
+              <a href="#">Product</a>
+              <a href="#">Vision</a>
+              <a href="#">Features</a>
+              <a href="#">About</a>
+              <a href="#">Contact</a>
+            </nav>
+        
+            <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+          </header>
 
-Digital Marketing
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
```

Comment:
It looks like the img is filling the width of the body rather than the width of the container.

Reply:
This is happening with the border-top and border-bottom dividers in your "services-main-content" class below as well. Take a look at your index.html page to see how the body and .container are being styled.